txclib/utils.py

```diff
@@ -282,6 +282,11 @@ def perform_single_request(method, url, headers, fields, manager, skip_decode,
     except HTTPError:
         logger.error("HTTP error")
         raise
+    except Exception as e:
+        logger.error(str(e))
```

Comment:
would be cool if the user can request a more verbose logging (for troubleshooting)

Reply:
@kouk I don't think that stack traces are relevant here.

There are two possible ways for this new code block to be executed:
Either the server responded with an error (and the response message will be printed), or something went wrong with our code and an exception was raised.

An example stack trace of the first case is the following:
```Exception in thread Thread-20:
Traceback (most recent call last):
  File "/usr/local/Cellar/python/2.7.14/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/usr/local/Cellar/python/2.7.14/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 754, in run
    self.__target(*self.__args, **self.__kwargs)
  File ".../txclib/utils.py", line 279, in perform_single_request
    r_value = parse_tx_response(response, skip_decode)
  File ".../txclib/utils.py", line 320, in parse_tx_response
    raise Exception("Error received from server: {}".format(data))
Exception: Error received from server: Could not import file: Invalid plural types for string: month:%(duration)d months. Language supports: ['one', 'few', 'many', 'other'], but found: ['one', 'few', 'many'] instead.
```
...which doesn't look helpful to me.

In the second case, would it be any helpful for a user to see a stack trace from our code that might be a bug? If so, we can use the already present `--traceback` flag to print it, but it doesn't really make any sense to me.